tsconfig.json

```diff
@@ -15,7 +15,9 @@
     "rootDir": "./src",
     "strict": true,
     "esModuleInterop": true,
-    "noImplicitAny": true
+//changes to handle build errors 
+//    "noImplicitAny": true
+    "noImplicitAny": false
```

Comment:
What build errors were you running into?  I would like to correct the errors vs turning this off.

Reply:
> What build errors were you running into? I would like to correct the errors vs turning this off.

Completely agree if another method would solve this.   Below is the error I received.   I also found this link to others receiving this error when creating new plug-in builds on homebridge 1.6... https://github.com/homebridge/HAP-NodeJS/issues/998 though the workaround they suggested didn't work when I tried to just create a declarations file, but there is probably a way I needed to refer to that declaration file.
__________________________________
pi@homebridge:~/development/homebridge-aqua-connect-lite $ npm run build

> homebridge-aqua-connect-lite@1.2.2 build
> rimraf ./dist && tsc

node_modules/homebridge/node_modules/hap-nodejs/dist/lib/Advertiser.d.ts:5:29 - error TS7016: Could not find a declaration file for module '@homebridge/dbus-native'. '/home/pi/development/homebridge-aqua-connect-lite/node_modules/homebridge/node_modules/@homebridge/dbus-native/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/homebridge__dbus-native` if it exists or add a new declaration (.d.ts) file containing `declare module '@homebridge/dbus-native';`

5 import { InvokeError } from "@homebridge/dbus-native";
                              ~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in node_modules/homebridge/node_modules/hap-nodejs/dist/lib/Advertiser.d.ts:5
_____________________________________